lbrynet/daemon/Downloader.py

```diff
@@ -89,8 +88,7 @@ def check_status(self):
         else:
             log.debug("Waiting for stream descriptor (%i seconds)", self.timeout_counter)
 
-    def convert_max_fee(self):
-        currency, amount = self.max_key_fee['currency'], self.max_key_fee['amount']
+    def convert_max_fee(self, currency, amount):
         return self.exchange_rate_manager.convert_currency(currency, "LBC", amount)
```

Comment:
Why these changes?

Reply:
I had something else in mind at the time but based on an earlier comment of yours, I would say that this change would allow for the method's use for `converted_fee_amount`.